docs/content/en/release-notes/_index.md

```diff
@@ -57,9 +57,168 @@ The K8ssandra helm chart deploys the following components. Some are optional, an
 Operators are software extensions to Kubernetes that make use of custom resources to manage applications and their components. Thus, for example, "Reaper Operator" deploys and configures Reaper. "Reaper" itself manages the actual Cassandra repair operations. Similarly, "Prometheus Operator" deploys and configures Prometheus. "Prometheus" itself manages the actual collection of relevant OS / Cassandra metrics. "Medusa Operator" configures and orchestrates the backup and restore operations. "Medusa" itself runs the container that performs backups of Cassandra data. 
 {{% /alert %}}
 
-## Upgrade notice
+## Upgrade notices
 
 {{% alert title="Important!" color="warning" %}}
+Before upgrading to K8ssandra 1.3.0, be sure to read the sections below.
+{{% /alert %}}
+
+### Upgrading to K8ssandra 1.3.0 and Cassandra 4.0
+
+When you upgrade from a prior K8ssandra release to 1.3.0, the default `cassandra.version` is now `4.0.0`. A changed `num_tokens` default may prevent Cassandra 4.0 from starting. This section explains how to avoid the issue. 
+
+First, some background information. The default value for `num_tokens` in Cassandra 3.11 is 256. The default in Cassandra 4.0 is 16. The `num_tokens` setting defines the number of tokens randomly assigned to this node on the ring. The more tokens, relative to other nodes, the larger the proportion of data that this node will store.
+
+The issue: [Upgrading from Cassandra 3.11 to 4.0 fails if num_tokens is not set](https://github.com/k8ssandra/k8ssandra/issues/1029). As noted for this scenario, the Cassandra logs will include the following error, and Cassandra will not start:
+
+```
+org.apache.cassandra.exceptions.ConfigurationException: Cannot change the number of tokens from 256 to 16
+```
+
+To avoid the issue, first check the `num_tokens` settings in your K8ssandra-deployed Cassandra 3.11 database. Let's walk through an example.
+
+#### Initial installation
+
+This example starts by installing K8ssandra 1.3.0 with Cassandra 3.11.10 and the latter's default `num_tokens` (256). 
+
+```bash
+helm install k8ssandra -f k8ssandra.values.yml k8ssandra/k8ssandra
+```
+
+k8ssandra.values.yml:
+
+```yaml
+cassandra:
+  version: "3.11.10"
+  datacenters:
+    - 
+      name: dc1
+      size: 1
+stargate:
```

Comment:
You can probably just take the stargate part of the example here, then you don't need to include the note like I mentioned in my comments earlier.  It'll remove a little bit of confusion probably.

Reply:
Will do